### ISSUE TITLE
Accessibility Feedback for Design Intent: Preferred Medical Facilities

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22466]
## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** Josh

## Notes

I reviewed these wires with @andaleliz ahead of the meeting and have left comments in UXPin.
Main points include:

- Make sure the desktop version of edit mode has a `<legend>` and is contained within a `<fieldset>`
- Heads up to developers that safari + VO can act strange with linebreaks in radio options, we may have to use `aria-label` to make sure the location is announced
- Focus should be circular when going between edit and read modes
- "+ 2 more facilities" should be a button, upon pressing it, focus should be sent to the newest item in the list

**Comment by 1Copenut** [comment]
@joshkimux I agree with all of your feedback here. Wanted to ask about this one

> "+ 2 more facilities" should be a button, upon pressing it, focus should be sent to the newest item in the list

Would a live region that had `aria-relevant="additions" aria-atomic="false"` make sense here? If it worked correctly, it'd read out the additional facilities and we wouldn't have to move focus off the Show N more button. This is a tricky one, but thought I'd mention it as an option.

**Comment by joshkimux** [comment]
@1Copenut Let's bring it up during the review! My only possible concern would be around verbosity if there's +5 facilities being added at once. This would be fun to codepen out to test 😄

**Comment by sshein** [comment]
just learned "codepen out" is a verb ;) <3

**Comment by andaleliz** [comment]
Thanks @joshkimux and @1Copenut ! I updated the desktop mock-ups to include the fieldset legend, and replied to comments in the UXPin (and made corresponding updates).  Appreciate your feedback!

**Comment by andaleliz** [comment]
@joshkimux coming back to this for a quick question:

>Heads up to developers that safari + VO can act strange with linebreaks in radio options, we may have to use aria-label to make sure the location is announced

Is this the case with checkbox options too? I assume so but want to be sure since we're using checkboxes instead of of radios.

**Comment by joshkimux** [comment]
@andaleliz Yes, that's correct 👍
